tests/main/base-none/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check that a snap with base "none" can be installed
+
+execute: |
+    #shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+
+    echo "Install a snap with 'base: none'"
+
+    snap pack "$TESTSLIB/snaps/test-snapd-base-none"
+    snap install --dangerous test-snapd-base-none_1.0_all.snap
```

Comment:
Should probably check a scenario in which we try to install a snap with base: none and some apps.

Reply:
Added.